demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py

```diff
@@ -723,9 +723,9 @@ def test_having_two_file_paths_not_same_pack(repo, mocker, first_file_content, s
 
 
 @pytest.mark.parametrize('known_words_content, expected_known_words',
-                         [(['[known_words]', 'wordament'], ['wordament']),
-                          (['[known_words]'], []),
-                          ([], [])])
+                         [(['[known_words]', 'wordament'], ['test_pack', 'wordament']),
+                          (['[known_words]'], ['test_pack']),
+                          ([], ['test_pack'])])
 def test_find_known_words_from_pack(repo, known_words_content, expected_known_words):
```

Comment:
Lets add some kind of documentation for this feature in the doc of this test so the expectation doesn't look suspicious